requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
```

Comment:
Please remove all those requirements changes, they're not needed anymore.

Reply:
Changes reverted